tests/load-tests/load-tests.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Let's add "example" somewhere in the name?

Reply:
Sure